packages/zwave-js/src/lib/node/Node.ts

```diff
@@ -5896,9 +5896,72 @@ ${formatRouteHealthCheckSummary(this.id, otherNode.id, summary)}`,
 			} catch {
 				return false;
 			}
-		} else {
-			// No way to set the time
-			return false;
+		} else if (
+			timeAPI.isSupported() &&
+			timeAPI.supportsCommand(TimeCommand.DateReport) &&
+			timeAPI.supportsCommand(TimeCommand.TimeReport)
```

Comment:
Would there ever be a case where Time CC is supported, but these two commands aren't?  It looks like they're required in both versions of Time CC.